### PR TITLE
PAAS-1929 interpret error stack as one line

### PR DESCRIPTION
**Pull request by laurentfufu**
JIRA issue: https://jira.jahia.org/browse/PAAS-1929

Short description:

### CODE REVIEW

File: packages/jcustomer/migrations/migrate-to-v10.yaml [hunk 1]
```diff
@@ -21,10 +21,12 @@ onInstall:
   - checkEsClusterStatus
   - checkElasticsearchDatadogCustomChecks: es
   - eventsUpdate
+  - setGlobalRepoRootUrl
   ### End Pre-checks
 
   - removeJettyIODebug                      # PAAS-1896
   - removeOperatingMode                     # PAAS-1920
+  - multilineDDConf                         #
```
Comment (kevinvandel) [inline packages/jcustomer/migrations/migrate-to-v10.yaml:29]: this hashtag is very mysterious...

Comment (amitc-jahia) [inline packages/jcustomer/migrations/migrate-to-v10.yaml:29]: Add the ticket ID at the end to be consistent as other actions.

File: packages/jcustomer/migrations/migrate-to-v10.yaml [hunk 2]
```diff
@@ -51,3 +53,22 @@ actions:
...
+  multilineDDConf:
+    - cmd[cp]: |-
+        # md5sum of file without the password line
+        newDDconfMD5="4d7ca78ebdbda5ef1e93fa44b2ee87bb"
+        confFile=/etc/datadog-agent/conf.d/jmx.d/conf.yaml
+        actualConfMD5=$(sed -r '/^\s+password:/d' $confFile | md5sum | awk '{print $1}')
+        if [ "$newDDconfMD5" = "$actualConfMD5" ]; then
+          exit 0
+        fi
+        pwd=$(awk '/^\s+password:\s*/ {print $NF; exit}' /etc/datadog-agent/conf.d/jmx.d/conf.yaml)
```
Comment (amitc-jahia) [inline packages/jcustomer/migrations/migrate-to-v10.yaml:66]: Can use the var `confFile` instead of the full path.

**Review by amitc-jahia — CHANGES_REQUESTED** [review]

File: packages/jcustomer/migrations/migrate-to-v10.yaml (hunk 1 above)
Comment (laurentfufu) [reply packages/jcustomer/migrations/migrate-to-v10.yaml]: yeah sorry for that :grin: 
fixed!

Comment (laurentfufu) [reply packages/jcustomer/migrations/migrate-to-v10.yaml]: yeah sorry for that :grin: 
fixed!

File: packages/jcustomer/migrations/migrate-to-v10.yaml (hunk 2 above)
Comment (laurentfufu) [reply packages/jcustomer/migrations/migrate-to-v10.yaml]: I missed this one indeed, fixed!

**Review by amitc-jahia — APPROVED** [review]

**Pull request merged**